tests/lib/tools/tests.pkgs.apt.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/sh
+
+remap_one() {
+    case "$1" in
+        man)
+            if [ "$(cat /etc/os-release && echo "$ID")" = debian ]; then
+                echo "man-db"
+            else
+                echo "$1"
+            fi
+            ;;
+        printer-driver-cups-pdf)
+            if [ "$(cat /etc/os-release && echo "$ID")" = debian ] || [ "$(cat /etc/os-release && echo "$ID/$ID_VERSION")" = ubuntu/14.04 ]; then
```

Comment:
Same question

Reply:
This is from https://github.com/snapcore/snapd/pull/9479/files#diff-6eea9b76e2bf27f60519f7b5c98333c9R14 - ubuntu 14.04 re-mapped this package to `cups` pdf 

I see I got the condition backwards though :) I'll correct this (it was late)